benchmark/utils/general.hpp

```diff
@@ -434,4 +465,250 @@ gko::remove_complex<ValueType> compute_max_relative_norm2(
 }
 
 
+/**
+ * A class for controlling the number warmup and timed iterations.
+ *
+ * The behavior is determined by the following flags
+ * - 'repetitions' switch between fixed and adaptive number of iterations
+ * - 'warmup' warmup iterations, applies in fixed and adaptive case
+ * - 'min_repetitions' minimal number of repetitions (adaptive case)
+ * - 'max_repetitions' maximal number of repetitions (adaptive case)
+ * - 'min_runtime' minimal total runtime (adaptive case)
+ * - 'repetitions_growth_factor' controls the increase between two successive
```

Comment:
nit
```suggestion
 * - 'repetition_growth_factor' controls the increase between two successive
```
or change the Gflags name